PlainTasks.py

```diff
@@ -833,3 +837,101 @@ def run(self, edit):
                 msg = msg.replace(o, r)
 
         sublime.set_clipboard(msg)
+
+class PlainTasksArchiveOrgCommand(PlainTasksBase):
+    def runCommand(self, edit):
+        # Archive the curent subtree to our archive file, not just completed tasks.
+        # For now, it's mapped to ctrl-shift-o or super-shift-o
+
+        # TODO: Mark any tasks found as complete, or maybe warn.
+
+        # Get our archive filename
+        archive_filename = self.__createArchiveFilename()
+
+        # Figure out our subtree
+        region = self.__findCurrentSubtree()
+        if region.empty():
+            # How can we get here?
+            sublime.error_message("Error:\n\nCould not find a tree to archive.")
+            return
+
+        # Write our region or our archive file
+        success = self.__writeArchive(archive_filename, region)
+
+        # only erase our region if the write was successful
+        if success:
+            self.view.erase(edit,region)
+
+        return
+
+    def __writeArchive(self, filename, region):
+        # Write out the given region
+
+        sublime.status_message('Archiving tree to {}'.format(filename))
```

Comment:
`sublime.status_message(u'Archiving tree to {0}'.format(filename))` to support ST2, and all other strings should start with `u` and contain index for format.